packages/experimental/Badge/src/PresenceBadge/PresenceBadge.styling.ts

```diff
@@ -24,28 +24,25 @@ export const stylingSettings: UseStylingOptions<PresenceBadgeProps, PresenceBadg
   slotProps: {
     root: buildProps(
       (tokens: PresenceBadgeTokens, theme: Theme) => {
-        const { width, height, borderWidth, right, bottom } = tokens;
+        const { width, height, borderWidth } = tokens;
         const borderGap = borderWidth * 2;
         return {
           style: {
             ...getBadgePosition(tokens),
             width: width + borderGap,
             height: height + borderGap,
-            display: 'flex',
             alignItems: 'center',
             flexDirection: 'row',
             alignSelf: 'flex-start',
             justifyContent: 'center',
             position: 'absolute',
             backgroundColor: tokens.backgroundColor,
             ...borderStyles.from(tokens, theme),
-            ...layoutStyles.from(tokens, theme),
-            right,
-            bottom,
+            paddingHorizontal: tokens.paddingHorizontal,
           },
         };
       },
-      ['backgroundColor', 'width', 'height', 'bottom', 'right', 'top', 'left', ...borderStyles.keys, ...layoutStyles.keys],
+      ['backgroundColor', 'width', 'height', 'bottom', 'right', 'top', 'left', 'borderWidth', 'paddingHorizontal', ...borderStyles.keys],
```

Comment:
I don't think you need borderWidth, that should be included in borderStyles

Reply:
yeah, thanks for catching this